src/platform/plugins/private/inspect_component/server/routes/component_data/get_component_data.test.ts

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { sep } from 'path';
+import { REPO_ROOT } from '@kbn/repo-info';
+import type { KibanaRequest } from '@kbn/core/server';
+import { httpResourcesMock } from '@kbn/core/server/mocks';
+import { loggingSystemMock } from '@kbn/core/public/mocks';
+import { getComponentData } from './get_component_data';
+import { getComponentCodeowners } from '../../lib/codeowners/get_component_codeowners';
+import type { GetComponentDataRequestBody } from './get_component_data';
+
+jest.mock('../../lib/codeowners/get_component_codeowners');
+
+describe('getComponentData', () => {
+  const mockRequest = {
+    body: {
+      path: `${REPO_ROOT}${sep}src${sep}test-file.tsx`,
```

Comment:
This might read better by using the `join` util from the path module here and skip the `sep` util, so you'd have something like so; 

```
path: join(REPO_ROOT, 'src', 'test-file.tsx')
```

or if you want to keep the `sep` util; 

```
path: [REPO_ROOT, 'src', 'test-file.tsx'].join(sep)
```